src/language/utils.ts

```diff
@@ -125,6 +125,14 @@ export function someAncestor(node: ts.Node, predicate: (n: ts.Node) => boolean):
     return predicate(node) || (node.parent && someAncestor(node.parent, predicate));
 }
 
+export function isAssignment(node: ts.Node) {
+    if (node.kind === ts.SyntaxKind.BinaryExpression) {
+        const binaryExpression = node as ts.BinaryExpression;
+        return binaryExpression.operatorToken.kind >= ts.SyntaxKind.FirstAssignment
+            && binaryExpression.operatorToken.kind <= ts.SyntaxKind.LastAssignment;
+    }
```

Comment:
I feel like this function should always produce a return value, i.e. have an `else { return false; }` at the end.